server/src/models/Project.ts

```diff
@@ -41,6 +40,16 @@ export class Project extends BaseModel {
 	@IsString()
 	files: string;
 
+	@Column()
+	@Field()
+	@IsString()
+	environment: string;
+
+	@Column()
+	@Field()
+	@IsString()
+	main: string;
```

Comment:
main is so generic, we can use **mainFile** or **activeFile**.

Reply:
yes for mainFile, you right
activeFile is something else btw

Reply:
i will make the modifications in last PR so i won't have to rebase branches